src/components/VHeader/VSearchBar/VClearButton.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <VIconButton
+    :button-props="{ variant: 'plain' }"
+    :icon-props="{ iconPath: closeIcon }"
+    size="smaller"
+    class="border-tx text-dark-charcoal-70"
+    :aria-label="$t('browse-page.search-form.clear')"
```

Comment:
```vue
    :aria-label="$t('browse-page.search-form.clear')"
```

Curiously this doesn't appear to be working:

<img width="286" alt="CleanShot 2022-10-04 at 08 39 00@2x" src="https://user-images.githubusercontent.com/6351754/193821503-efc24c50-d5e5-42b9-b0fb-ccb31cd284a0.png">

Reply:
I forgot to add the string to `en.json` in this branch 😬